components/connection-token-handler/sample-cr.yaml

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
Do we need this sample? If yes - shouldn't it be placed somewhere near the CR and not inside controller's code?

Reply:
No we don't, removed :)